ckanext/openafrica/blueprint.py

```diff
@@ -0,0 +1,42 @@
+from flask import Blueprint
+from ckan.plugins.toolkit import render
+
+openafrica = Blueprint('openafrica', __name__)
+
+def toc():
+    return render('home/about/toc.html')
+
+def accessibility():
+    return render('home/about/accessibility.html')
+
+def coc():
+    return render('home/about/coc.html')
+
+def moderation():
+    return render('home/about/moderation.html')
+
+def faq():
+    return render('home/about/faq.html')
+
+def privacy():
+    return render('home/about/privacy.html')
+
+def contact():
+    return render('home/about/contact.html')
+
+def atlas():
+    return render('atlas.html')
+
+rules = [
+    ("/about/terms-and-conditions", "toc", toc),
```

Comment:
Unless I'm missing something, a closure would work better here:
```python
def static_path(path):
    def render_path():
        return render(path)

    return render_path
```

And we'd use it as:
```python
rules = [
    ("/about/terms-and-conditions", "toc", static_path('home/about/toc.html')),
    ("/about/accessibility", "accessibility", static_path('home/about/accessibility.html')),
    ....
```

1. May be you can think of a better, shorter name that static_path,
2. Don't mix use of single and double quotes; pick one and sick with it.